### DISCUSSION TITLE
URL bar

### DISCUSSION THREAD

**Discussion by ghost** [protz/thunderbird-conversations#1621] [Q&A] [answered]
Hi,

how I can remove the URL bar on the topside?

![image](https://user-images.githubusercontent.com/12504756/113403149-7adba000-93a6-11eb-8a80-470165d3968e.png)

Thanks!

**Comment by Standard8** [comment, marked as answer]
Unfortunately you can't. That is a side effect of using the newer WebExtensions APIs which are better for us in the long term.

Talking about the long term, we may be able to be in a position of replacing the message display properly, at which point we might be able to drop the urlbar again, but for now that's not really viable.
